gammapy/image/profile.py

```diff
@@ -194,6 +196,241 @@ def save(self, filename):
         raise NotImplementedError
 
 
+class ImageProfile(object):
+    """
+    Image profile class.
+
+    The image profile data is stored in `~astropy.table.Table` object, with the
```

Comment:
I'd suggest to put a `Parameters` section, even if there's just one parameter in `__init__`.
Keeping the table column description in the docstring is fine, I think.